tests/test_base.cpp

```diff
@@ -562,7 +580,7 @@ std::string server_node::config_path() const
 	return m_path;
 }
 
-server_config server_node::config() const
+server_config &server_node::config()
```

Comment:
From test_base.hpp:
```c++
class config_data:
...
	// it is used to prevent serialization of this object to a config (json)
	bool m_serializable;
```
server_config contains vector of backends of config_data type.
It is possible to specify backend, but it won't be serialized to a ioserv test config.
This feature used in backend_test (see test_enable_backend_after_config_change()).